src/lab14.py

```diff
@@ -644,78 +644,8 @@ def matches(self, node):
     def __repr__(self):
         return "PseudoclassSelector({}, {})".format(self.pseudoclass, self.base)
 
+@wbetools.patch(CSSParser)
 class CSSParser:
-    def __init__(self, s):
-        self.s = s
-        self.i = 0
-
-    def whitespace(self):
-        while self.i < len(self.s) and self.s[self.i].isspace():
-            self.i += 1
-
-    def literal(self, literal):
-        if not (self.i < len(self.s) and self.s[self.i] == literal):
-            raise Exception("Parsing error")
-        self.i += 1
-
-    def word(self):
-        start = self.i
-        in_quote = False
-        while self.i < len(self.s):
-            cur = self.s[self.i]
-            if cur == "'":
-                in_quote = not in_quote
-            if cur == "(":
-                in_parens = True
```

Comment:
These two lines were the only difference from `lab13.py`'s `word` and note that the `in_parens` variable is not used anywhere else, so I think this method is identical to `lab13` and doesn't need to be restored.